apps/launch.html.markerb

```diff
@@ -7,73 +7,86 @@ order: 10
 ...
 ```
 
-The flyctl Django scanner has taken ownership of my launch. Visit our [Django guide](/docs/django/getting-started/) to see how that story will end. (Spoiler: it has a happy ending.)
+The flyctl Django scanner has taken ownership of the launch. You can tweak the basic settings on the Fly Launch web page and then run 'fly deploy' to deploy the new app. Visit our [Django guide](/docs/django/getting-started/) to see how that story will end. (Spoiler: it has a happy ending.)
 
 ## Custom launch
 
 You can nudge `fly launch` to better suit your project. 
 
-### Choose how the Docker image is built
+### Point to an image or use a Dockerfile to build
 
 Tell `fly launch` how you want to get the Docker image for your app, using either the `--image` or `--dockerfile` option, or by catching the Dockerfile launch scanner's attention with the presence of a [Dockerfile](/docs/languages-and-frameworks/dockerfile/) in your project source directory. The Dockerfile scanner doesn't do a lot of configuration, but it prevents other scanners from taking over.
 
 The actual Docker image build (or image pull) for a Fly App takes place during deployment. `fly launch` sets the stage by recording how to build, or get, the image, and both the first and all later deploys use that information.
 
 ### Customize the configuration file
 
-You can provide your own `fly.toml` and `fly launch` will offer to read it in, in lieu of the default config. `fly.toml` sets a starting point for the app configuration, and in some cases a framework launch scanner might overwrite parts of it.
+You can provide your own `fly.toml` and `fly launch` will offer to copy that configuration to a new app. `fly.toml` sets a starting point for the app configuration, and in some cases a framework launch scanner might overwrite parts of it.
 
 There are also [a number of other options](/docs/flyctl/launch/) you can use to exert control over `fly launch`.
 
 If `fly launch` doesn't have a scanner that can set up your app automatically, it will still initialize a new Fly App in your Fly.io organization and provide you with a default app configuration that's a reasonable starting point for a simple web app. 
 
-It's also possible to perform an entirely manual "launch", skipping all the launch scanners and full-service resource provisioning, using `fly apps create`, a hand-crafted (or copied) `fly.toml`, and step-by-step resource provisioning, followed by `fly deploy`.
+You can also perform an entirely manual "launch", skipping all the launch scanners and full-service resource provisioning, using `fly apps create`, a hand-crafted (or copied) `fly.toml`, and step-by-step resource provisioning, followed by `fly deploy`.
 
 ## After `fly launch`
 
-If you've run `fly launch` but haven't deployed yet, you can:
+If ran `fly launch` but didn't deploy yet (hint: you can do this with `fly launch --no-deploy`), or you deployed but want to make changes, then you can:
```

Comment:
Is there a convention to replace the present perfect with the past simple in this kind of doc? I believe it's less correct in generic English.

Reply:
I questioned this too, and obviously i forgot the "you". But there is no convention. Will look at it again.